fw/face/link-service.go

```diff
@@ -226,7 +226,7 @@ func (l *linkServiceBase) dispatchInterest(pkt *defn.Pkt) {
 	}
 
 	// Store name for easy access
-	pkt.Name = pkt.L3.Interest.NameV
+	pkt.Name = pkt.L3.Interest.Name()
```

Comment:
FYI: The goal of the Interest interface is to separate encoding details from high level semantics, since the spec of packet kept changing in those years.
Since the `LinkService` class needs to access LP packets, it has to know the low-level details and will be affected by spec change anyway.
So both interface method and field access are OK.

Reply:
My earlier thought was to re-implement a separate set of TLV types `FwInterest` and `FwData` that implement the `ndn.Interest` and `ndn.Data` interfaces, but only parse the fields needed by forwarding. (e.g. no need to get `Data.Content` or `Interest.AppParams` or any signature fields). If the forwarder uses the interface, this would be a drop-in replacement.

But this idea is on ice for now, doesn't seem to be the bottleneck.